index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Portfolio _one</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" crossorigin />
+    <script src="https://kit.fontawesome.com/fe8431d094.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <header>
+        <nav class="buttons_nav">
+            <h2 class="logo"><a href="#">My Logo</a></h2>
+            <a href="#" class="menu-icon"><img src="images/humburger.png" alt=""><i class="my logo"></i></a>
```

Comment:
- Hey, as this is the hamburger icon, it should not be wrapped in the `a` tag as this is not a link. Also, since you have an image already, kindly remove the `i` with the class of "my-logo" to improve the document structure of your page.